src/components/OrgSettings/AgendaItemCategories/OrganizationAgendaCategory.module.css

```diff
@@ -13,16 +13,32 @@
   gap: 10px;
 }
 
-.btnsContainer .btnsBlock {
-  display: flex;
-  gap: 10px;
+.btnsContainer .input {
+  /* Styling for the input field */
+  flex: 1;
+  min-width: 18rem;
+  position: relative;
 }
 
-.btnsContainer button {
-  display: flex;
-  align-items: center;
+.btnsContainer input {
+  /* Styling for the input border */
+  outline: 1px solid var(--bs-gray-400);
 }
 
+.btnsContainer .input button {
+  /* Styling for the button in the input field */
+  width: 52px;
+}
+
+.inputField {
+  background-color: white;
+  box-shadow: 0 1px 1px #31bb6b;
```

Comment:
1. We are migrating away from the green theme as part of our support for color blind users.
2. Take cues from other similar elements in this file and the PR mentioned in the issue.
    1. https://github.com/PalisadoesFoundation/talawa-admin/pull/2466